ProxyAppsSuite/tests/ember.yaml

```diff
@@ -0,0 +1,470 @@
+                files: halo3d.out
+            results_from_rank:
+                regex: '\#\sResults\sfrom\srank\:\s+(\d+)'
+                action: store
+                files: halo3d.out
+            pex:
+                regex: '\#\sProcessor\sGrid\:\s+(\d+)'
+                action: store
+                files: halo3d.out
+            pey:
+                regex: '\#\sProcessor\sGrid\:\s+\d+\sx\s+(\d+)\sx\s+\d+'
+                action: store
+                files: halo3d.out
+            pez:
+                regex: '\#\sProcessor\sGrid\:\s+\d+\sx\s+\d+\sx\s+(\d+)'
+                action: store
+                files: halo3d.out
+            nx:
+                regex: '#\sData\sGrid\s\(per\srank\):\s+(\d+)'
+                action: store
+                files: halo3d.out
+            ny:
+                regex: '#\sData\sGrid\s\(per\srank\):\s+\d+\sx\s+(\d+)\sx\s+\d+'
+                action: store
+                files: halo3d.out
+            nz:
+                regex: '\#\sData\sGrid\s\(per\srank\):\s+\d+\sx\s+\d+\sx\s+(\d+)'
+                action: store
+                files: halo3d.out
+            iterations:
+                regex: '\#\sIterations\:\s+(\d+)'
+                action: store
+                files: halo3d.out
+            variables:
+                regex: '\#\sVariables\:\s+(\d+)'
+                action: store
+                files: halo3d.out
+            sleep:
+                regex: '\#\sSleep\:\s+(\d+)'
+                action: store
+                files: halo3d.out
+
+
+
+incast:
+    inherits_from: _no_build
+    summary: Multiple inbound messages, I/O-like
+    run:
+        cmds:
+            - cd mpi/incast
+            - srun -n 4 ./incast -iterations 500000 -msgsize 100000 &> incast.out
+            - cat incast.out
+            - mv incast.out ../../incast.out
+    result_parse:
+        regex:
+            passed:
+                regex: '\#\s+Incast\s+Ranks\:\s+(\d+)\s*'
+                action: 'store_true'
+                files: incast.out
+            incast_ranks:
+                regex: '\#\s+Incast\s+Ranks\:\s+(\d+)\s*'
+                action: store
+                files: incast.out
+            iterations:
+                regex: '\#\s+Iterations\:\s+(\d+)\s*'
+                action: store
+                files: incast.out
+            message_size:
+                regex: '\#\s+Message\s+Size\:\s+(\d+)\s*'
+                action: store
+                files: incast.out
+            time_taken:
+                regex: '\ + (\d+\S+)\ +\d+\S+\ +\d+\S+'
+                action: store
+                files: incast.out
+            msgs_per_s:
+                regex: '\ + \d+\S+\ +(\d+\S+)\ +\d+\S+'
+                action: store
+                files: incast.out
+            mb_per_second:
+                regex: '\ + \d+\S+\ +\d+\S+\ +(\d+\S+)'
+                action: store
+                files: incast.out
+
+pingpong:
+    inherits_from: _no_build
```

Comment:
I'm starting to think that we should get rid of the _no_build/_build approach, and just build it everywhere... may make it simpler. (sorry!)